src/test/java/com/open/boot/core/utils/CommonUtilTest.java

```diff
@@ -0,0 +1,107 @@
+package com.open.boot.core.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+
+public class CommonUtilTest {
+
+    @Test
+    public void testIsEmpty() {
+        Assert.assertTrue(CommonUtil.isEmpty(""));
+        Assert.assertTrue(CommonUtil.isEmpty(null));
+
+        Assert.assertFalse(CommonUtil.isEmpty("foo"));
+    }
+
+    @Test
+    public void testGetStringSimple() {
+        Assert.assertEquals("foob***obar",
+                CommonUtil.getStringSimple("foobar"));
+    }
+
+    @Test
+    public void testGetYestoday() {
+        Date date =
+                new Date(System.currentTimeMillis() - 24 * 60 * 60 * 1000);
+        String data = new SimpleDateFormat("yyyyMMdd").format(date);
+        Assert.assertEquals(data, CommonUtil.getYestoday());
+    }
+
+    @Test
+    public void testGetCurrentDate() {
+        Date date = new Date();
+        String data = new SimpleDateFormat("yyyyMMdd").format(date);
+        Assert.assertEquals(data, CommonUtil.getCurrentDate());
+    }
+
+    @Test
+    public void testGetCurrentTime() {
+        Date date = new Date();
+        String data = new SimpleDateFormat("yyyyMMddHHmmss").format(date);
+        Assert.assertEquals(data, CommonUtil.getCurrentTime());
+    }
+
+    @Test
+    public void testGetGenderByIdNumber() {
+        Assert.assertEquals(1,
```

Comment:
Add tests that return `0` and `2` (were any tests found by Cover for those branches?)

Reply:
Done